<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessingWindow</class>
 <widget class="QDialog" name="ProcessingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Processing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Processing...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="SpinnerWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinnerWidget</class>
   <extends>QWidget</extends>
   <header>gui/components/spinnerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
